casual/casual-caller/src/test/groovy/se/laz/casual/connection/caller/CasualCallerImplTest.groovy

```diff
@@ -317,27 +304,25 @@ class CasualCallerImplTest extends Specification
    def 'TPNOTRAN tpacall in transaction'()
```

Comment:
Again, weird name for this test now that it is rewritten and doesn't use TPNOTRAN flag